docs/content/python/new-roles/install.md

```diff
@@ -34,10 +38,8 @@ this would be helpful to you!)
   * macOS
   * Windows
 
-{{% callout "Have feedback?" "green" %}}
+## Have feedback?
 
 Have feedback on this documentation or the library itself? It's under
```

Comment:
Need to replace this text everywhere with:

If at any point you get stuck, drop into our Slack (join slack link) or send an email (mailto engineering@) to our engineering team and we'll unblock you.